site2/docs/developers-landing.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: developers-landing
 title: Pulsar for Developers
-sidebar_label: "get started"
+sidebar_label: ""
 ---
 
 Developing applications for Pulsar can be a fun and rewarding experience. With Pulsar, you can quickly create, deploy, and manage your services using a powerful CLI tool and a comprehensive set of libraries. The topics below will get you started!
 
-- [Simulation tools](develop-tools.md)
-- [Developing binary protocol](developing-binary-protocol.md)
-- [Modular load manager](develop-load-manager.md)
-- [Pulsar plugin development](develop-plugin.md)
+- [Simulation tools](develop-tools)

```

Comment:
`.md` is required for internal file links. Need to add them back for all the items.